ivy/utils/backend/sub_backend_handler.py

```diff
@@ -200,3 +210,65 @@ def find_available_sub_backends(sub_backends_loc):
             available_sub_backends.append(sub_backend)
 
     return available_sub_backends
+
+
+def _find_available_sub_backend_implementations(sub_backends):
+    result = dict()
+    for sub in sub_backends:
+        sub_backend = ivy.utils.dynamic_import.import_module(_sub_backend_dict[sub])
+        for k, v in sub_backend.__dict__.items():
+            if isinstance(v, Callable) and not k.startswith("__"):
+                result[k] = result.get(k, []) + [sub]
+
+    return result
+
+
+def available_sub_backend_implementations(obj: Union[Callable, str]) -> list:
```

Comment:
I think it would be fine to only accept a `fn_name: str` instead which would be the function name to avoid complicating the implementation

Reply:
I did this to avoid the user passing something like ` ivy.available_sub_backend_implementations("__name__")`

Reply:
I think we can just rename this to `fn_name` which should tell the user what we expect in that case!